pxtcompiler/emitter/snippets.ts

```diff
@@ -311,9 +311,13 @@ namespace ts.pxtc.service {
                 if (typeNode.kind === SK.StringKeyword || deflKind === SK.StringKeyword) {
                     return paramDefl.indexOf(`"`) != 0 ? `"${paramDefl}"` : paramDefl;
                 }
-
+                let type = checker && checker.getTypeAtLocation(param);
+                const typeSymbol = getPxtSymbolFromTsSymbol(type.symbol, apis, checker);
```

Comment:
The line above would indicate that checker could be undefined, which would cause it to get a null deref here

```suggestion
                const typeSymbol = getPxtSymbolFromTsSymbol(type?.symbol, apis, checker);
```